### DISCUSSION TITLE
about Authenticating the CLI

### DISCUSSION THREAD

**Discussion by yoyoLee6** [fonoster/yaps#490] [Q&A]
err：
7 PERMISSION_DENIED: permission denied
Accessing endpoint api.fonoster.io... Invalid credentials or endpoint

please tell me how to solve the problem
![Uploading 微信图片_20230328115919.png…]()

**Comment by psanders** [comment]
I see there is png file but I can't see which command are you running. 

Which credentials are you passing? Are you connecting to your own instance of Fonoster?

**Comment by psanders** [comment]
I will close this conversation to maintain organization. However, if you need further assistance in the future, feel free to re-open it.
